azure-pipelines.yml

```diff
@@ -203,6 +203,32 @@ stages:
           pathtoPublish: wheelhouse
           artifactName: wheels
 
+  - job: macos_universal2
+    pool:
+      vmImage: 'macOS-12'
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.10'
```

Comment:
I seemingly tried but failed to only have a python `3.10` wheel created. :shrug: .